include/tudat/astro/observation_models/observationModel.h

```diff
@@ -316,24 +316,27 @@ class ObservationModel
             std::vector< Eigen::Matrix< double, 6, 1 > >& linkEndStates,
             const std::shared_ptr< ObservationAncilliarySimulationSettings< TimeType > > ancilliarySetings = nullptr )
     {
+        // Add time bias if necessary
+        TimeType observationTime = computeBiasedObservationTime( time );
```

Comment:
Should the conversion to a biased observation time not be done after checking the validity of the reference link ends? (if this second step is removed, see below, there should be no harm in keeping this here)